include/geode/mesh/core/edged_curve.h

```diff
@@ -54,9 +56,9 @@ namespace geode
 
         const Point< dimension >& point( index_t vertex_id ) const;
 
-        double edge_length( index_t edge_id ) const;
+        double length( index_t edge_id ) const;

```

Comment:
Not this one, we should revert this change

Reply:
why this one and not `barycenter`?